utilities_help/createToolboxMLTBX.m

```diff
@@ -5,6 +5,7 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
+% Copyright 2023 The MathWorks, Inc.
 prjFileName=fullfile("fsdaToolboxPackaging.prj");
```

Comment:
Why are we doing this? Since, 
prjFileName=fullfile("fsdaToolboxPackaging.prj")
should return: "fsdaToolboxPackaging.prj"
In that case can we remove this?

Reply:
Removed fullfile in line no. 8.